app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const { getTopics } = require("./connectors/topics.connector");
+
+const app = express();
+app.use(express.json())
```

Comment:
Don't need app.use(express.json()) until we are using it for a patch/post etc. Best practice is generally to not to commit unnecessary code - something to bear in mind going forwards